react-native/src/blueTheme.ts

```diff
@@ -6,10 +6,10 @@ All rights reserved.
 This code is licensed under the BSD-3 license found in the LICENSE file in the root directory of this source tree and at https://opensource.org/licenses/BSD-3-Clause.
 **/
 
-//@ts-ignore
 import {blue, red, gray, lightBlue, white} from '@pxblue/colors';
+import { Theme as PXBlueTheme } from '@pxblue/react-native-components/core/theme';
```

Comment:
I would suggest for the time being that we duplicate the theme Type definition here inside the react-native folder instead of creating a peer dependency. Since we publish all of the themes in a single package (for now), it doesn't make sense to force someone doing an angular project to install our react native component library just to get a type definition that they don't need.